test/e2e/test_broker_pubsub.go

```diff
@@ -65,12 +66,22 @@ func BrokerWithPubSubChannelTestImpl(t *testing.T) {
 	client.Core.CreateRBACResourcesForBrokers()
 	client.Core.CreateBrokerOrFail(brokerName, lib.ChannelTypeMeta)
 
+	// Create the Istio ServiceEntry.
+	istioServiceEntry := resources.IstioServiceEntry(
```

Comment:
Ohh I see you are using it here still

Reply:
You are right, we don't have to create it per namespace. 
I installed it from the Shell script and removed the namespace creation, and it still works :)